packages/javascript/bh-shared-ui/src/components/ExploreTable/ExploreTable.tsx

```diff
@@ -14,31 +14,111 @@
+    const labelsMap = makeFormattedObjectInfoFieldsMap(firstItem);
+
+    const initialColumns: ColumnDef<any, any>[] = [
+        {
+            accessorKey: '',
+            id: 'action-menu',
+            cell: () => (
+                <button className='pl-4'>
+                    <FontAwesomeIcon icon={faEllipsis} className='rotate-90 dark:text-neutral-light-1' />
+                </button>
+            ),
+        },
+        {
+            accessorKey: 'nonTierZeroPrincipal',
+            header: () => {
+                return <span className='dark:text-neutral-light-1'>Non Tier Zero Principal</span>;
+            },
+            cell: ({ row }) => {
+                return (
+                    <div className='flex justify-center items-center relative'>
+                        <NodeIcon nodeType={row?.original?.nodetype || 'N/A'} />
+                    </div>
+                );
+            },
+        },
+    ];
+
+    const columns: ColumnDef<any, any>[] = useMemo(
+        () =>
+            firstItem
+                ? // If column order exists in redux/localStorage, use that
+                  Object.keys(firstItem).map((key: any) => {
+                      return {
+                          accessorKey: key,
+                          header: labelsMap?.[key]?.label || capitalize(key),
+                          cell: (info: any) => String(info.getValue()),
+                          id: key,
+                          size: 150,
+                      } as ColumnDef<any, any>;
+                  })
+                : [],
+        [labelsMap, firstItem]
+    );
 
-    if (!open) return null;
+    if (!open || !data) return null;
 
+    const finalColumns = [...initialColumns, ...columns];
     return (
-        <div className='absolute bottom-4 left-4 right-4 h-1/2 bg-pink-300 flex justify-center items-center'>
-            <div className={cn({ hidden: openManageColumns })}>
-                <Button onClick={toggleOpenManageColumns}>Manage Columns</Button>
-                <Button onClick={onClose}>CLOSE</Button>
+        <div
+            className={`border-2 overflow-hidden absolute z-10 bottom-16 left-4 right-4 max-h-1/2 h-[475px] bg-neutral-light-2`}>
+            <div className='explore-table-container w-full h-full'>
+                <TableControls
+                    onDownloadClick={() => console.log('download icon clicked')}
+                    onExpandClick={() => console.log('expand icon clicked')}
+                    onManageColumnsClick={() => console.log('manage columns button clicked')}
+                    onCloseClick={onClose}
+                    tableName='Results'
+                    resultsCount={mungedData?.length}
+                    SearchInputProps={{
+                        onChange: (e) => setSearchInput(e.target.value),
+                        value: searchInput,
+                        placeholder: 'Search',
+                    }}
```

Comment:
Something about this is causing the search input to update really slowly. The display is to spec (which is the intent of this PR) so when we go to wire up these other callbacks, we should take a minute to dive into the search input!